environs/manual/linux/provisioner.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 Canonical Ltd.
+// Copyright 2016 Cloudbase Solutions SRL
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package linux
+
+import (
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/environs/manual/common"
+
+	"github.com/juju/loggo"
+)
+
+var (
+	logger = loggo.GetLogger("juju.environs.manual.linux")
+)
+
+const Scope = "ssh"
+
+type provision struct {
```

Comment:
`provisioner`? We have `NewProvisioner()`, and generally structs are nouns, not verbs.

Why is this a struct? It has no member variables and only one method. Can we just have a function instead?